example/screens/ConnectWithRoomManagerScreen.tsx

```diff
@@ -0,0 +1,152 @@
+  Text,
+  View,
+} from 'react-native';
+import {connectScreenLabels} from '../types/ComponentLabels';
+import {useJellyfishClient} from '@jellyfish-dev/react-native-client-sdk';
+import {Button, TextInput, DismissKeyboard} from '../components';
+import {NativeStackScreenProps} from '@react-navigation/native-stack';
+import {AppRootStackParamList} from '../navigators/AppNavigator';
+import type {BottomTabScreenProps} from '@react-navigation/bottom-tabs';
+import {TabParamList} from '../navigators/AppNavigator';
+import {CompositeScreenProps} from '@react-navigation/native';
+import {usePermissionCheck} from '../hooks/usePermissionCheck';
+
+type Props = CompositeScreenProps<
+  BottomTabScreenProps<TabParamList, 'ConnectWithRoomManager'>,
+  NativeStackScreenProps<AppRootStackParamList>
+>;
+
+const {URL_INPUT, TOKEN_INPUT, CONNECT_BUTTON} = connectScreenLabels;
+
+async function getJellyFishServer(
+  roomManagerUrl: string,
+  roomName: string,
+  userName: string,
+) {
+  const url = roomManagerUrl.endsWith('/')
+    ? roomManagerUrl
+    : roomManagerUrl + '/';
+  const result = await fetch(
+    `${url}rooms/${roomName.trim()}/users/${userName.trim()}`,
+  );
+
+  const tokenData = (await result.json()) as {
+    url: string;
+    token: string;
+  };
+
+  return {
+    jellyfishUrl: tokenData.url,
+    token: tokenData.token,
+  };
+}
+
+const ConnectScreen = ({navigation}: Props) => {
+  const {connect} = useJellyfishClient();
+  const [connectionError, setConnectionError] = useState<string | undefined>(
+    undefined,
+  );
+  const [loading, setLoading] = useState(false);
+
+  const [roomManagerUrl, setRoomManagerUrl] = useState(
+    process.env.ROOM_MANAGER_URL ?? '',
+  );
+  const [roomName, setRoomName] = useState('');
+  const [userName, setUserName] = useState('');
+
+  usePermissionCheck();
+
+  const onTapConnectButton = async () => {
+    try {
+      setConnectionError(undefined);
+      setLoading(true);
+      const {jellyfishUrl, token} = await getJellyFishServer(
+        roomManagerUrl,
+        roomName,
+        userName,
+      );
+
+      await connect(jellyfishUrl, token);
+
+      navigation.navigate('Preview');
+    } catch (e) {
+      const message =
+        'message' in (e as Error) ? (e as Error).message : 'Unknown error';
+      setConnectionError(message);
+    } finally {
+      setLoading(false);
+    }
+  };
+
+  return (
+    <DismissKeyboard>
+      <SafeAreaView style={styles.safeArea}>
+        <View style={styles.container}>
+          {connectionError && (
+            <Text style={styles.errorMessage}>{connectionError}</Text>
+          )}
+          <Image
+            style={styles.logo}
+            source={require('../assets/jellyfish-logo.png')}
+            resizeMode="contain"
+          />
+          <TextInput
+            onChangeText={setRoomManagerUrl}
+            value={roomManagerUrl}
+            accessibilityLabel={URL_INPUT}
+            placeholder="Room Manager URL"
```

Comment:
Imo somewhere should be specified how to obtain this url (eg in app's readme.) and its format  (eg `<protocol>://<ip_address>:<port>`

Reply:
This part will be added once we finish `js-server-sdk`